specs_internal/PLL.yaml

```diff
@@ -46,7 +46,7 @@
 name: PLL
 desc: Phase locked loop
 category: periphery
-timescale: 1ps/1ps
+timescale: 1fs/1fs
```

Comment:
@ashrafnisar @abdulhameed-akram please watch for potential simulation failures with different timescale